.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
 .DS_Store
 /.idea
-.env
\ No newline at end of file
+.env
```

Comment:
Why did this line change? Did you add and extra space?

Reply:
![image](https://user-images.githubusercontent.com/8351695/79452775-434fd880-801b-11ea-91c6-8b92a86131e2.png)
No extra space, just added a new line to this file.

Reply:
Can you add a final newline to fix the diff in the future?

Reply:
The line 4 is not changing now